src/components/src/main/java/org/apache/jmeter/visualizers/backend/graphite/GraphiteBackendListenerClient.java

```diff
@@ -149,24 +146,20 @@ public void run() {
         sendMetrics();
     }
 
-    /**
-     * Send metrics to Graphite
-     */
+    /** Send metrics to Graphite */
     protected void sendMetrics() {
         // Need to convert millis to seconds for Graphite
         long timestampInSeconds = TimeUnit.SECONDS.convert(System.currentTimeMillis(), TimeUnit.MILLISECONDS);
         synchronized (LOCK) {
-            for (Map.Entry<String, SamplerMetric> entry : getMetricsPerSampler().entrySet()) {
-                final String key = entry.getKey();
-                final SamplerMetric metric = entry.getValue();
-                if(key.equals(CUMULATED_METRICS)) {
+            getMetricsPerSampler().forEach((key, metric) -> {
```

Comment:
Did you check if use of lambda here would impact negatively performances ? (As per other PR discussion ?)

Thanks